cluster-up/cluster/kind-sriov/provider.sh

```diff
@@ -21,8 +21,11 @@ function print_available_nics() {
 }
 
 function set_kind_params() {
-    export KIND_VERSION="${KIND_VERSION:-0.18.0}"
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.27.1@sha256:9915f5629ef4d29f35b478e819249e89cfaffcbfeebda4324e5c01d53d937b09}"
+    version=$(cat cluster-up/cluster/$KUBEVIRT_PROVIDER/version)
```

Comment:
nit: Using `read` to read the first line would be nicer, but this is not that important.